README.md

```diff
@@ -6,9 +6,6 @@ http://www.stoimen.com/blog/2010/06/11/friday-algorithms-quicksort-difference-be
 
 https://khan4019.github.io/front-end-Interview-Questions/sort.html#quickSort
 
-
```

Comment:
Not bad. Don't do this next time tho. Cheers!![bitmoji](https://render.bitstrips.com/v2/cpanel/7216901-120523106_6-s4-v1.png?transparent=1&palette=1&width=246)